src/plugins/filedelete/filedelete.cpp

```diff
@@ -178,8 +198,11 @@ static void extract_ca_file(filedelete* f, drakvuf_t drakvuf, vmi_instance_t vmi
     if ( test != (test2 * 4096) )
         return;
 
+    static int sequence_number = 0;
```

Comment:
This would need to be defined as a member of the filedelete class, the use of globals is something we are trying to avoid in the plugins.

Reply:
OK. The same problem is with the set of modified handles.

And I missed that file handles does not unique among multiple processes in the system, right?

Reply:
I do it on Monday.